org.ekstep.sunbirdcommonheader-1.5/editor/headerApp.js

```diff
@@ -59,6 +58,56 @@ angular.module('org.ekstep.sunbirdcommonheader:app', ["Scope.safeApply", "yaru22
     $scope.isFlagReviewer = false;
     $scope.editorEnv = "";
     $scope.showEditMeta = true;
+    $scope.contentCredits = [];
+    $scope.listLimit = 5;
+    /*
+    * Add owner details and update current count with new values.
+    */
+    $scope.updateOwnershipList = function(event, nodeData) {
+        var contentCredit = {'ownedBy':nodeData.data.metadata.identifier, 
+                                'owner':nodeData.data.metadata.creator, 
+                                'ownershipType': nodeData.data.metadata.contentType };
+        
+        var index = _.findIndex($scope.contentCredits, function(o) {
+                            return o.owner == nodeData.data.metadata.creator; });
+        if(nodeData.data.metadata.creator && index == -1) {
+            $scope.contentCredits.push(contentCredit);
+        }
+    }
+
+    /*
+    * Remove owern details and update new owner details values.
+    */
+    $scope.removeOwnershipList = function(event, nodeData) {
+        $scope.contentCredits = [];
+        var rootNode = ecEditor.jQuery("#collection-tree").fancytree("getRootNode").getFirstChild();
+        rootNode.visit(function(node) {
+            var contentCredit = {'ownedBy':node.data.metadata.identifier, 
+                                    'owner':node.data.metadata.creator, 
+                                    'ownershipType': node.data.metadata.contentType };
+
+            var index = _.findIndex($scope.contentCredits, function(o) {
+                    return o.owner == node.data.metadata.creator; });
+            if(node.data.metadata.creator && index == -1) {
+                $scope.contentCredits.push(contentCredit);
+            }
+        });
+    }
+
+    /*
+    * Initialize listLimit value when click on user icon button.
+    */
+    $scope.showContributionList = function() {
+        $scope.listLimit = 5;
+    };
+
+    /*
+    * Increase size of listLimit when click on see more button.
+    */
+    $scope.addListSize = function() {
```

Comment:
Why are you showing only +5 when we click on showmore?

Show entire list with scrollbar when it is more than 10.